blank_rust_project/src/main.js

```diff
@@ -62,14 +62,29 @@ function signedInFlow() {
     // Displaying the signed in flow container.
     document.getElementById('signed-in-flow').classList.remove('d-none');
 
-    window.contract.welcome({account_id:window.accountId}).then(response => document.getElementById('speech').innerText = response.text);
+    getGreeting();
 
     // Adding an event to a sign-out button.
     document.getElementById('sign-out-button').addEventListener('click', () => {
         walletAccount.signOut();
         // Forcing redirect.
         window.location.replace(window.location.origin + window.location.pathname);
     });
+
+    // Adding an event to change greeting button.
+    document.getElementById('change-greeting').addEventListener('click', () => {
+        setGreeting();
+    });
+}
+
+async function setGreeting() {
+    await window.contract.set_greeting({message:'Howdy'});
+    getGreeting();
+}
+
+async function getGreeting() {
+    const response = await window.contract.welcome({account_id:window.accountId});
```

Comment:
`getGreeting` is inconsistent with the `.welcome` call. Can we align those?

Reply:
done